### PR TITLE
UIOR-493 Display the PO Line record when a new PO Line is added

### PR DESCRIPTION
**Pull request by AndreiShumski2**
## Purpose
https://issues.folio.org/browse/UIOR-493

## Pre-Merge Checklist
Before merging this PR, please go through the following list and take appropriate actions.
- [x] I've added appropriate record to the CHANGELOG.md
- Does this PR meet or exceed the expected quality standards?
  - [x] Code coverage on new code is 80% or greater
  - [x] Duplications on new code is 3% or less
  - [x] There are no major code smells or security issues
- Does this introduce breaking changes?
  - [ ] If any API-related changes - okapi interfaces and permissions are reviewed/changed correspondingly
  - [x] There are no breaking changes in this PR.

If there are breaking changes, please **STOP** and consider the following:

- What other modules will these changes impact?
- Do JIRAs exist to update the impacted modules?
  - [ ] If not, please create them
  - [ ] Do they contain the appropriate level of detail?  Which endpoints/schemas changed, etc.
  - [ ] Do they have all they appropriate links to blocked/related issues?
- Are the JIRAs under active development?
  - [ ] If not, contact the project's PO and make sure they're aware of the urgency.
- Do PRs exist for these changes?
  - [ ] If so, have they been approved?

Ideally all of the PRs involved in breaking changes would be merged in the same day to avoid breaking the folio-testing environment.  Communication is paramount if that is to be achieved, especially as the number of intermodule and inter-team dependencies increase.

While it's helpful for reviewers to help identify potential problems, ensuring that it's safe to merge is ultimately the responsibility of the PR assignee.

### CODE REVIEW

File: test/bigtest/tests/POLine/line-create-test.js [hunk 1]
```diff
@@ -49,11 +49,12 @@ describe('Create PO Line simple test', function () {
         await lineEditPage.quantityPhysical.fill(2);
         await lineEditPage.physicalCreateInventory.select('None');
         await lineEditPage.saveButton.click();
-        await orderDetailsPage.whenLoaded();
+        await ordersList.whenListLoaded();
```
Comment (aliaksei-chumakou) [inline test/bigtest/tests/POLine/line-create-test.js:52]: why orders list? in ticket mentioned that line details pane should be opened

Comment (AndreiShumski2) [reply test/bigtest/tests/POLine/line-create-test.js:52]: Form is closed and OrdersList with Line details are presented

Comment (aliaksei-chumakou) [reply test/bigtest/tests/POLine/line-create-test.js:52]: so what is the point of this test if it doesn't test ticket's requirement?

File: src/components/LayerCollection/LayerPOLine.js [hunk 2]
```diff
@@ -167,14 +167,19 @@ function LayerPOLine({
...
-      .then(() => openOrder(saveAndOpen))
-      .then(() => {
+      .then(({ id: poLineId }) => openOrder(saveAndOpen, poLineId))
+      .then(lineIdResponse => {
         sendCallout({
           message: <SafeHTMLMessage id="ui-orders.line.create.success" />,
           type: 'success',
         });
+
+        const pathname = lineIdResponse
```
Comment (aliaksei-chumakou) [inline src/components/LayerCollection/LayerPOLine.js:177]: Could you please explain the reason or result for this `pathname` ? I can't get it

File: src/components/LayerCollection/LayerPOLine.js [hunk 3]
```diff
@@ -128,7 +128,7 @@ function LayerPOLine({
   );
 
   const openOrder = useCallback(
-    (saveAndOpen) => {
+    (saveAndOpen, poLineId) => {
```
Comment (aliaksei-chumakou) [inline src/components/LayerCollection/LayerPOLine.js:131]: this is not belong here, it's simply not used in this function

File: src/components/LayerCollection/LayerPOLine.js (hunk 2 above)
Comment (AndreiShumski2) [reply src/components/LayerCollection/LayerPOLine.js:177]: The idea is that if user just create new POL, after saving he will see new POL details. But if he clicks save and open order button he will be redirected to PO details

Comment (AndreiShumski2) [reply src/components/LayerCollection/LayerPOLine.js]: agree

File: src/components/LayerCollection/LayerPOLine.js (hunk 3 above)
Comment (AndreiShumski2) [reply src/components/LayerCollection/LayerPOLine.js]: fixed

File: test/bigtest/tests/POLine/line-create-test.js [hunk 4]
```diff
@@ -43,17 +43,24 @@ describe('Create PO Line simple test', function () {
...
         await lineEditPage.selectOrderFormat('Physical resource');
         await lineEditPage.listUnitPrice.fill(3.333);
         await lineEditPage.quantityPhysical.fill(2);
         await lineEditPage.physicalCreateInventory.select('None');
         await lineEditPage.saveButton.click();
-        await orderDetailsPage.whenLoaded();
+
+        const newLine = this.server.schema.lines.first();
+
+        purchaseOrder.update({ ...purchaseOrder.attrs, compositePoLines: [newLine.attrs] });
```
Comment (aliaksei-chumakou) [inline test/bigtest/tests/POLine/line-create-test.js:57]: what is the reason of  `...purchaseOrder.attrs` ? compositePoLines: [newLine.attrs] should be enough

File: test/bigtest/tests/POLine/line-create-test.js [hunk 5]
```diff
@@ -43,17 +43,24 @@ describe('Create PO Line simple test', function () {
 
     describe('Fill values and click save', () => {
       beforeEach(async function () {
+        const purchaseOrder = this.server.schema.orders.first();
```
Comment (aliaksei-chumakou) [inline test/bigtest/tests/POLine/line-create-test.js:46]: this const could be together with the rest, no need to declare it here

File: test/bigtest/tests/POLine/line-create-test.js (hunk 1 above)
Comment (AndreiShumski2) [reply test/bigtest/tests/POLine/line-create-test.js]: Appreciate your support with test

**Pull request merged**